### ISSUE TITLE
license link is dead

### ISSUE THREAD

**Issue by NikVolf** [ErisBlastar/cplusequality#10]
though i am not like forcing link to lead to a proper license
just a humble assumption that it might probably do so

**Comment by ErisBlastar** [comment]
Yep it's dead, I tried CPR to revive it, but it is still dead. It's dead Jim! It is pining for the Fjords!

**Comment by Arinerron** [comment]
https://github.com/ErisBlastar/cplusequality/blob/master/LICENSE

**Comment by luismasuelli** [comment]
I loved the referente to Monthy Python
